provisioning/transport/mqtt/src/ProvisioningTransportHandlerMqtt.cs

```diff
@@ -56,14 +56,7 @@ public class ProvisioningTransportHandlerMqtt : ProvisioningTransportHandler
             TransportFallbackType transportFallbackType = TransportFallbackType.TcpWithWebSocketFallback)
         {
             FallbackType = transportFallbackType;
-            if (FallbackType == TransportFallbackType.WebSocketOnly)
-            {
-                Port = WsPort;
-            }
-            else
-            {
-                Port = MqttTcpPort;
-            }
+            Port = FallbackType == TransportFallbackType.WebSocketOnly ? WsPort : MqttTcpPort;
```

Comment:
why is the above line needed? can we just not use transportFallbackType in the comparison?

Reply:
I don't understand your question here; are you asking why we set the port or why we have this comparison?

Reply:
I think my question is not valid. Not able to see the full context of the code in the PR confused me. FallbackType is not another local variable. So nothing to change here.